File: test7C#/test7/Model/Pizza/MeatPizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model.Pizza
{
    class MeatPizza : SimplePizza
    {
        public MeatPizza() : base("Meat Pizza", Convert.ToDecimal(29.99))
        {
            base.Topping(base.Topping() + ",ground beef,pepperoni");
        }
    }
}

File: test7Go/oop/models/pizza/MeatPizza.go
package pizza

import (
	"fmt"
)

type MeatPizza struct {
	SimplePizza
}

func NewMeatPizza() MeatPizza {
	simplePizza := NewSimplePizza("Meat Pizza", 29.99)
	simplePizza.Topping = fmt.Sprintf("%s,ground beef,pepperoni", simplePizza.Topping)

	return MeatPizza{simplePizza}
}
